### ISSUE TITLE
TypeError: IMask is not a constructor

### ISSUE THREAD

**Issue by severeon** [uNmAnNeR/imaskjs#56]
Starting this morning, when I use the vue-imask component I get "TypeError: IMask is not a constructor".

**Comment by severeon** [comment]
I am able to work around the issue by creating a local shim / webpack alias. 

`imask-shim.js`

```js
require('imask/dist/imask.es.js')

export default window.IMask
```

and I added the following to my webpack config:

```js

    resolve: {
      alias: {
        'imask$': path.resolve('./src/imask-shim.js')
      }
    },
```

**Comment by uNmAnNeR** [comment]
Thanks for issue.
The problem in using `rollup-multi-entry-plugin` for building IMask. It should be avoided if default exports is used. Fixed in v3.2.4.
Probably i have to add at least one test for each plugin to check integration succeeds.
